test/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 # Enable the tests
 
+# Allow the source files to find headers in src/
+include_directories(${PROJECT_SOURCE_DIR}/src)
```

Comment:
Actually, yes. I was using SleepForMilliseconds in my test in order to get appropriate timings for each time unit.